.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```

Comment:
I would tailor these to the dependencies of the selfhost repo. We don't really care what their browser version is.

Reply:
Ok would you say adding the OS (Linux or MacOS) would be helpful and then asking which cloud provider they are using? 

So something like:
**OS:**
- Linux
- MacOS

**Cloud Provider**
- AWS
- Google Cloud
- DigitalOcean
- Other

Reply:
Yep these would be super great. LGTM!

Reply:
Ok I added them!